### DISCUSSION TITLE
Plain HTML output?

### DISCUSSION THREAD

**Discussion by tomByrer** [BuilderIO/jsx-lite#1476] [Ideas]
Is there a way to output 'plain' HTML output?  I thought about using Mitosis as a SSG.
Or is there an existing output that is very close to HTML & would need only a tiny bit of modification?

**Comment by samijaber** [comment]
Yes, we have an HTML generator. See this screenshot (using the old playground, because we haven't added the HTML generator as an option to our new playground just yet)

![CleanShot 2024-06-10 at 11 53 02@2x](https://github.com/BuilderIO/mitosis/assets/1393142/3f069c95-179e-4226-9903-44cf3c5c5606)

**Reply by tomByrer** [reply]
TY
I thought there was one; I wish the old playground was left up for reasons like this.

**Reply by samijaber** [reply]
We are working on migrating all missing features to the new playground!
